examples/pinot-async.py

```diff
@@ -0,0 +1,55 @@
+import asyncio
+
+import httpx
+from pinotdb import connect
+
+
+async def foo():
+    async with connect(host='localhost', port=8099, path='/query/sql',
+                       scheme='https', verify_ssl=False, use_async=True) as conn:
+        curs = await conn.execute_async("""
+            SELECT count(*)
+              FROM foo
+             LIMIT 10
+        """)
+        for row in curs:
+            print(row)
+
+    conn = connect(
+        host='localhost', port=8099, path='/query/sql', scheme='https',
+        verify_ssl=False, use_async=True)
+
+    curs = conn.cursor()
+    await curs.execute_async("""
+        SELECT count(*)
+          FROM foo
+         LIMIT 10
+    """)
+    for row in curs:
+        print(row)
```

Comment:
maybe we can use an actual overlapping async example. like burst fire 10 queries and then awaits them:
```
async_futures = []
for batch in range (10):
  async_futures.append(curs.execute_async(
    "SELECT COUNT(*) FROM foo LIMIT 10, " + batch * 10)

for async_future in async_futures:
  await async_future;
  for row in curs:
    print(row)
```